wled00/FX_2Dfcn.cpp

```diff
@@ -378,14 +375,14 @@ void Segment::blur2D(uint8_t blur_amount, bool smear) {
       uint32_t part = color_fade(cur, seep);
       curnew = color_fade(cur, keep);
       if (y > 0) {
-        if (carryover) curnew = color_add(curnew, carryover, true);
-        uint32_t prev = color_add(lastnew, part, true);      
+        if (carryover) curnew = color_add(curnew, carryover);
+        uint32_t prev = color_add(lastnew, part);
```

Comment:
Maybe I overlooked something in the previous discussions - I guess that setting the last parameter to true was done on purpose ... are you sure it does the same thing?

Reply:
this change is to be approved. It does not do 100% the same thing but it does it faster. I tested this with fire2012 at full blur and the 'overflow' happens only once (i.e. for one pixel) every few frames. There may be situations where this change actually is visible but I could not find one (except the ones I updated in the other PR, smearing looks better without saturation)